### ISSUE TITLE
[Bug]: showOpenDialog with openDirectory *STILL* can not choose folder

### ISSUE THREAD

**Issue by devzzzero** [electron/electron#44909]
### Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

electron@33.2.1 | MIT | deps: 3 | versions: 1353

### What operating system(s) are you using?

Other Linux

### Operating System Version

openSuse 15.5 LTS

### What arch are you using?

x64

### Last Known Working Electron version

???

### Expected Behavior

[Bug]: showOpenDialog with openDirectory can not choose folder #30125

Hi. This bug is a lot like #30125 except on linux with a recent electron (33.?) 

So here is my code snippet from my `main.ts` file 

```
    'openDir' : ipcMain.handle('openDir', async (ev, { title, defaultPath } )=>{
      console.log(`MAIN.openDir ${title} ${defaultPath}`);
      const thePath =  await dialog.showOpenDialog({
        title,
        properties:['openDirectory'],
      });
      console.log(`MAIN.openDir ${title} ${defaultPath} ${thePath}`);
      return thePath;
    }),
    'openFile' : ipcMain.handle('openFile', async (ev, { title, defaultPath } )=>{
      console.log(`MAIN.openFile ${title} ${defaultPath}`);
      const thePath = await dialog.showOpenDialog({
        title, defaultPath,
        properties:['openFile', 'showHiddenFiles'],
      });
      console.log(`MAIN.openFile ${title} ${defaultPath} got ${JSON.stringify(thePath)}`);
      return thePath;
    }),
```

### Actual Behavior

`openFile` IPC/showopendialog works.

However, when I use `openDir` IPC,  the KDialog (linux) can not be closed when I am navigating to a directory.
Even worse, even though the 'openDirectory' properties are clearly set, it can only be 'non-cancel' closed unless I select a **file!**
I don't think this is a KDE Dialog issue. 
Furthermore, I purposefully mistyped "openDirectory", but nary a peep from electron.

**Comment by nikwen** [comment]
Electron version is still supported. Version parsing failed. Should be fixed by #44918.

**Comment by devzzzero** [comment]
apology for the noise. this bug is invalid.
